### ISSUE TITLE
Rich text editor is double-spacing unless holding shift

### ISSUE THREAD

**Issue by philipt4** [laurent22/joplin#5745]
As someone who doesn't find much use for markdown editors, i find myself using the rich text editor part of joplin for my notetaking.
I find it very odd that i have to hold shift in order to begin writing on a new line as opposed to a new paragraph/double-space being made. 

Am i missing something or is this a thing that Joplin devs are firm on and don't want to allow an toggle option of some sort?

**Comment by ghost** [comment]
Can someone comment on if theres a way to prevent this from happening. I like Joplin a lot but I don't understand why it double spaces the lines. If for example I'm documenting a config file or python errors, if I copy and paste the content of the file into joplin it now is now twice as long too scroll though, and if I want to copy the text from joplin into an editor like nano , I have a terribly formatted config file I now have to go in and manually fix.

So basically I would like a way to prevent Joplin from restructuring content pasted into it.

**Comment by PauloGDPeixoto** [comment]
I recently started using Joplin and this issue is very annoying.

**Comment by mmertdemirhan** [comment]
I came across with the same issue and this worked for me:
Tools ⇾ Options ⇾ Appearance ⇾ Show Advanced Settings ⇾ Custom Stylesheet for rendered Markdown 
Open the note editor or your IDE and copy this:

p {margin:0; padding:0;}

Save it and restart Joplin. Hope works for you.

**Comment by ghost** [comment]
> I came across with the same issue and this worked for me: Tools ⇾ Options ⇾ Appearance ⇾ Show Advanced Settings ⇾ Custom Stylesheet for rendered Markdown Open the note editor or your IDE and copy this:
> 
> p {margin:0; padding:0;}
> 
> Save it and restart Joplin. Hope works for you.

I tried it and it didn't seem to have an effect. I copied a bunch of lines that had no space inbetween them and pasted it into Joplin and Joplin added the additional space inbetween each line.

**Comment by mmertdemirhan** [comment]
I found the solution on [reddit](
https://libredd.it/r/joplinapp/comments/pnpzo9/double_line_space_when_pressing_enter/) and other user said this code snippet worked for him:
p { margin-top: 1em ; margin-bottom: 0em ; }

**Comment by mmertdemirhan** [comment]
If these code snippets do not work, you can search the issue in [forum](https://discourse.joplinapp.org/)

**Comment by PauloGDPeixoto** [comment]
> I came across with the same issue and this worked for me: Tools ⇾ Options ⇾ Appearance ⇾ Show Advanced Settings ⇾ Custom Stylesheet for rendered Markdown Open the note editor or your IDE and copy this:
> 
> p {margin:0; padding:0;}
> 
> Save it and restart Joplin. Hope works for you.

This worked for me, thank you! I was getting a bit frustrated as I didn't notice that I had to restart Joplin for this custom userstyle to take effect. But after restarting, worked like a charm! Thank you!
